bundles/org.openhab.binding.melcloud/src/main/java/org/openhab/binding/melcloud/internal/api/json/ListDevicesResponse.java

```diff
@@ -0,0 +1,445 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.melcloud.internal.api.json;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link ListDevicesResponse} is responsible of JSON data For MELCloud API
+ * Response of Devices List.
+ * Generated with jsonschema2pojo
+ *
+ * @author Luca Calcaterra - Initial contribution
+ */
+public class ListDevicesResponse {
+
+    @SerializedName("ID")
```

Comment:
I didn't check all these fields, but GSON has an option `FieldNamingPolicy.UPPER_CAMEL_CASE` that converts the first character of a field to an uppercase. It looks like that could have been used here. Maybe not something to change for now. Just something to think of (if it might make it more readable).

Reply:
Didn't know this option, thanks for the info.

I think it can indeed be used throughout the project.
@lucacalcaterra, @paulianttila, do you have any preference, use `@SerializedName` or `FieldNamingPolicy.UPPER_CAMEL_CASE`?
Is it ok if I check the project and make the proposed change?